build/app.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/cbroglie/mustache"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	sigsYamlFile   = "sigs.yaml"
+	templateDir    = "build"
+	indexTemplate  = fmt.Sprintf("%s/sig_index.mustache", templateDir)
+	listTemplate   = fmt.Sprintf("%s/sig_list.mustache", templateDir)
+	footerTemplate = fmt.Sprintf("%s/footer.mustache", templateDir)
+	sigListOutput  = "sig-list.md"
+	sigIndexOutput = "README.md"
+)
+
+type Lead struct {
+	Name    string
+	Company string
+	GitHub  string
+}
+
+type Meeting struct {
+	Day       string
+	UTC       string
+	PST       string
+	Frequency string
+}
+
+type Contact struct {
+	Slack       string
+	MailingList string `yaml:"mailing_list"`
+	GitHubTeam  string `yaml:"github_team"`
+}
+
+type Sig struct {
+	Name              string
+	Dir               string
+	MissionStatement  string `yaml:"mission_statement"`
+	Leads             []Lead
+	Meetings          []Meeting
```

Comment:
`sig-scheduling` AFAIK has alternating meetings: biweekly Mondays 20:00 UTC and Wednesdays 07:30 UTC. I'm not sure how else we can express that than as a list of discrete meetings.